src/test/java/com/googleinterns/zoomtube/utils/TranscriptLineUtilTest.java

```diff
@@ -0,0 +1,103 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.googleinterns.zoomtube.utils;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.googleinterns.zoomtube.data.TranscriptLine;
+import java.io.IOException;
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.w3c.dom.Element;
+
+@RunWith(JUnit4.class)
+public final class TranscriptLineUtilTest {
+  @Rule public final MockitoRule mockito = MockitoJUnit.rule();
+  @Mock private Element node;
+  
+  private LocalDatastoreServiceTestConfig datastoreConfig =
+      (new LocalDatastoreServiceTestConfig()).setNoStorage(true);
```

Comment:
Can this be moved into the localServiceHelper?

Reply:
Moved!